framework/web/Request.php

```diff
@@ -944,6 +944,25 @@ protected function resolvePathInfo()
         return (string) $pathInfo;
     }
 
+    /**
+     * Encodes an ISO-8859-1 string to UTF-8
+     * @param string $s
+     * @return string the UTF-8 translation of <i>s</i>.
```

Comment:
I suggest to point source class by `@see` tag; also, I'm not sure about `<i>` - maybe markdown will work better?